src/components/NewCommentForm.tsx

```diff
@@ -43,26 +78,32 @@ export const NewCommentForm: React.FC = () => {
           <input
             type="text"
             name="email"
+            value={newComment?.email}
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': email })}
+            onChange={(event) => onCommentData(event, newComment)}
```

Comment:
Pass only the 'event' to the onCommentData function. Remove the second argument 'newComment'.